client/utils.ts

```diff
@@ -0,0 +1,22 @@
+export function safelyDefine(
+  elementName: string,
+  customElement: CustomElementConstructor
```

Comment:
I get an error here, maybe because `CustomElementConstructor` is not pulled in from anywhere?

Reply:
It is an interface from the `dom` library specified in the `tsconfig.json`, it shouldn't error for you 😞